pallets/ocex/src/lib.rs

```diff
@@ -360,12 +359,19 @@ pub mod pallet {
 			let trading_pair_info = TradingPairConfig {
 				base_asset: base,
 				quote_asset: quote,
-				min_price: min_order_price,
-				max_price: max_order_price,
-				price_tick_size,
-				min_qty: min_order_qty,
-				max_qty: max_order_qty,
-				qty_step_size,
+				min_price: Decimal::from(min_order_price.saturated_into::<u128>())
+					.div(&Decimal::from(UNIT_BALANCE)),
```

Comment:
Is it okay to use here potentially `panic`ing code? `.div()` can panic ([here](https://github.com/paupino/rust-decimal/blob/master/src/arithmetic_impls.rs#L218-L219)). I see that here used constant value (e.g. `UNIT_BALANCE`) but still...

Reply:
agree, two checks are required: wount overflow + not divided by zero prior to each op

Reply:
also interesting how 96 bit Decimal takes in 128 bit integer as saturated_into from very small and very big values, which potentially can underflow/overflow

Reply:
Resolved

Reply:
I can see that this was resolved for the `deposit()` method in [this](https://github.com/Polkadex-Substrate/Polkadex/pull/518) PR, but not here.

Reply:
Had discussion with Ivan, he explained. Resolving.